registrations/serializers.py

```diff
@@ -208,6 +208,15 @@ def validate(self, data):
                 ):
                     errors["date_of_birth"] = _("The participant is too old.")
 
+        if (
+            data.get("responsible_for_group") is False
```

Comment:
PEP8 states this should be written as: `not data.get("responsible_for_group")`

Reply:
`not data.get("responsible_for_group")` will falsely give the validation error if `responsible_for_group` is not given in the data on purpose

Reply:
Ahh, I buy that 🤝